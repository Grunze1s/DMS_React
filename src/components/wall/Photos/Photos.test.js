import React from 'react';
import Photos from './index';
import {
    configure,
    shallow
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})
const dummy_data={
    photoList:[{image:"a,bs"},{image:"sdfd"}],
    title:"photoTitle"
}

const setup = (props = {}, state = null) => {
    return (shallow( < Photos {...props}  />)
    )
}

describe('<Photos />', () => {
  

it("Photos render", () => { 
        const wrapper= setup({...dummy_data});
        expect(wrapper).toMatchSnapshot();
  
})
})

