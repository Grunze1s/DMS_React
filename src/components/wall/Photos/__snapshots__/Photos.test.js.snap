// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Photos /> Photos render 1`] = `ShallowWrapper {}`;
